client/app/components/routes/select-operator/form/AccessRequestReceived.tsx

```diff
@@ -44,20 +46,29 @@ export default async function AccessRequestReceived({
         Operation Representative has been received and will be reviewed.
       </p>
       {adminRequestJSX}
+      <Link href="/dashboard/select-operator">
+        <Button
+          className="my-10"
+          aria-label={"View Submitted Information"}
+          color="primary"
+          variant="contained"
+        >
+          View Submitted Information
+        </Button>
+      </Link>
```

Comment:
I don't feel super strongly about this and have likely done it myself before but wrapping a button in an `<a>` tag is not valid by HTML 5 specification:
https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element


I played around a bit and got it styled the same aside from the hover elements:
```
      <Link
        href="/dashboard/select-operator"
        aria-label={"View Submitted Information"}
        className="m-auto w-fit no-underline text-lg text-white py-[6px] px-[16px] rounded flex items-center bg-bc-primary-blue"
      >
        View Submitted Information
      </Link>{" "}
```

To make this cleaner and more reusable we could add a custom tailwind class using `@apply` in `globals.css` under the `.form-heading-label` class. Something like this: 

```
  .link-button-blue {
    @apply w-fit no-underline text-lg text-white py-[6px] px-[16px] rounded flex items-center bg-bc-primary-blue;
  }
```

The use on the link like this:

```
      <Link
        href="/dashboard/select-operator"
        aria-label={"View Submitted Information"}
        className="m-auto link-button-blue"
      >
        View Submitted Information
      </Link>{" "}
```

Reply:
Thanks, didn't know that about the <link><button /></link>. That pattern does show up a few times in the app (for example where I stole this code from haha), so I've made a tech debt ticket to fix it everywhere: https://github.com/bcgov/cas-registration/issues/784